testutil/testapp/x/testmodule/module.go

```diff
@@ -57,15 +57,14 @@ func (m *Module) InitGenesis(ctx sdk.Context, _ codec.JSONCodec, data json.RawMe
 			Power:  sdk.DefaultPowerReduction.Int64(),
 		},
 	}
-
 }
 
-func (m *Module) ExportGenesis(_ sdk.Context, _ codec.JSONCodec) json.RawMessage {
-	return []byte("{}")
+func (*Module) ExportGenesis(_ sdk.Context, _ codec.JSONCodec) json.RawMessage {
+	return json.RawMessage("{}")
 }
 
-func (m *Module) DefaultGenesis(_ codec.JSONCodec) json.RawMessage {
-	return []byte("{}")
+func (*Module) DefaultGenesis(_ codec.JSONCodec) json.RawMessage {
```

Comment:
aside: I'm not sure whether I've encountered unnamed receivers before. I guess they are generally rare because methods that don't rely on their receivers for anything may often be better suited as "independent" functions.

Reply:
Agreed. We only define the receiver here because we have to fulfill an external interface.